### ISSUE TITLE
Elementor and WooCommerce Storefront Theme - fonts/styles DON'T work

### ISSUE THREAD

**Issue by smile00000** [pojome/elementor#4768]
Hello,

Here in your Link:
https://elementor.com/create-ecommerce-website/

You discuss elementor and woocommerce storefront theme. I like using storefront bc I think its more reliable than astra and ocean etc themes..faster and better for woocommerce carts.

The only issue is that it ignores my fonts I choose in Elementor.  

Here is a video I made for you that shows this issue:
https://www.useloom.com/share/e5f7c13e05c34393915b7d8f65d98754

Can you PLEASE find a way in your code to FORCE bypass theme fonts and css so we can use exactly the fonts and styles of what is in elementor builder? 

Since you already have woocommerce products in elementor builder you should at least allow it to integrate with woocommerce storefront better. It's the fastest best theme for woocommerce and we need it to integrate with elementor.

Thanks in advance!

**Comment by smile00000** [comment]
What's really strange is that the footer fonts (the same fonts I want as the header) are working with elementor.

https://www.useloom.com/share/866f93742bc64c7aa01de241e7c31000

**Comment by smile00000** [comment]
So I've determined that it's the "navigation menu fonts" that are not working. Please advise.

https://www.useloom.com/share/7e777533c6e845c19cb28a1f63db4de2

**Comment by bainternet** [comment]
Thanks for reporting,
Please follow guidelines for opening a new issue/feature request, so we can help you better.
This report is missing key information: 

* Prerequisites.
* **Steps to replicate.**
* Isolating the problem.
* System info.

**Comment by smile00000** [comment]
I did provide steps to replicate. It's in the three videos I provided.


Cheers,

The Mood Machine
NYC/LA/Worldwide
(212) 951-1495
themoodmachine.com

**Comment by smile00000** [comment]
It's not a feature request your menu navigation widgets have a bug.

I did provide steps to replicate. It's in the three videos I provided.

Here's more details:

Step 1: Create a Navigation Menu Widget in Elementor.
Step 2: Change font to Poppins.
Step 3: You will notice that Poppins DOES NOT work in navigation menu links. It works everywhere else. AKA in my footer where it's just links, but not with your menu navigation widgets.

Isolating the problem: MENU LINK FONTS do NOT work with Elementor's menu navigation widgets & WooCommerce Storefront theme.

System Info: Most updated version of Wordpress & WooCommerce Storefront & Elementor Pro.
